src/node.cpp

```diff
@@ -1626,6 +1662,11 @@ LocalNode::~LocalNode()
 
     if (node)
     {
+        if (!sync->isUpSync())
+        {
+            node->setSyncable(false);
```

Comment:
should we set syncable true here if the sync is being shut down?  (the test for that is a few lines below)  thanks

Reply:
I dont think so. A remote node should retain its syncable property regardless of the current sync, no?